gensim/test/test_poincare.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Author: Jayant Jain <jayantjain1992@gmail.com>
+# Copyright (C) 2017 Radim Rehurek <me@radimrehurek.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Automated tests for checking the poincare module from the models package.
+"""
+
+import logging
+import unittest
+import os
+
+import numpy as np
+
+from gensim.models.poincare import PoincareData, PoincareModel
+
+
+module_path = os.path.dirname(__file__)  # needed because sample data files are located in the same folder
+datapath = lambda fname: os.path.join(module_path, 'test_data', fname)
```

Comment:
Please merge `develop` to you branch and replace this two lines to `from gensim.test.utils import datapath` (yesterday we refactored tests)